README.md

```diff
@@ -46,15 +46,15 @@ Tested disctributions, kernel versions:
 | Ports | 3 × USB-C, Mini-jack | ✔ Yes | All the ports work. Charging works only via left port. |
 | Graphic Dongle | USB-Typ-C to USB-Typ-A-/HDMI-/VGA | ✔ Yes | Works. HDMI monitor works, USB keyboard works. |
 | Keyboard |  | ⚠️ Minor issues | see [Keyboard details](#keyboard) below |
-| Touchpad | | ✔ Yes | Touchpad is detected and works good in GNOME. Left, right clicks, 2-finger scrolling, 2-finger zooming, 3-finger workspaces switching work. |
+| Touchpad | | ✔ Yes | Touchpad is an optional purchase. It works as intended and is fully functional. |
```

Comment:
This line is about touchpad, not the touchscreen. You could add the comment about the "optional purchase" to the "Display " section https://github.com/milkovsky/Linux-on-Lenovo-Slim-7-Carbon-AMD#touchscreen.

Reply:
Oh god, how could I oversee that. Thanks for the heads up *shame* :D

Reply:
Considering there are multiple ways to drive a touchscreen, can you confirm which way you use?

That differs from distro to distro. 
https://wiki.archlinux.org/title/Touchscreen

Reply:
No worries :)
I did not have to do any extra steps to setup touchscreen. It worked out-of-the-box for Ubuntu, PopOs, Fedora in my tests.